master/buildbot/test/integration/test_worker_comm.py

```diff
@@ -94,7 +94,12 @@ def remote_getWorkerInfo(self):
             'info': 'here',
             'worker_commands': {
                 'x': 1,
-            }
+            },
+            'numcpus': 1,
+            'none': None,
+            'os_release': b'\xe3\x83\x86\xe3\x82\xb9\xe3\x83\x88',
```

Comment:
didn't we say we should decode in the worker?

Reply:
I am 100% convinced that the master side will only get bytes and not string no matter how much I decode on worker.